docs/cas-server-documentation/configuration/Configuration-Properties.md

```diff
@@ -3527,6 +3527,17 @@ Provision and create established user profiles to identity stores.
 
 RESTful settings for this feature are available [here](Configuration-Properties-Common.html#restful-integrations) under the configuration key `cas.authn.pac4j.provisioning.rest`.
 
+### GitHub
+
+In addition to the [common block of settings](Configuration-Properties-Common.html#delegated-authentication-settings) , the following properties are additionally supported, when delegating authentication to GitHub:
```

Comment:
- Remove the preceding space. 
- Remove the spaces *before* commas, and only keep a single space after commas 
- Turn links to markdown links.
- Specify a blank line after "```". 
- End sentences with a period where applicable.